tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -26,6 +26,21 @@
 #import "CGContextInternal.h"
 #endif
 
+#ifdef TARGET_OS_MAC
+#define LOG_TEST_FILE(...)
```

Comment:
nah, not necessary to record the property in an xml file we don't generate for a platform we don't want to diff on :)